webpack/scenes/Content/ContentConfig.js

```diff
@@ -82,6 +83,58 @@ export default () => [
       pluralLabel: 'ostree_refs',
       singularLabel: 'ostree_ref',
     },
+    columnHeaders: [
+      { title: __('Name'), getProperty: unit => (<Link to={urlBuilder(`content/ostree_refs/${unit?.id}`, '')}>{unit?.name}</Link>) },
+      { title: __('Version'), getProperty: unit => unit?.version },
+    ],
+    tabs: [
+      {
+        tabKey: 'details',
+        title: __('Details'),
+        getContent: (contentType, id, tabKey) => <ContentInfo {...{ contentType, id, tabKey }} />,
+        columnHeaders: [
+          { title: __('Name'), getProperty: unit => unit?.name },
+          { title: __('Version'), getProperty: unit => unit?.version },
+          { title: __('Additional metadata'), getProperty: unit => unit?.additional_metadata },
```

Comment:
`additional_metadata` is a nested hash/object, so to use this you'd want to do something like `additional_metadata.sha256`. But in the case of OSTree,  additional_metadata is unused, so I would just remove this.

If you're curious what `additional_metadata` is, look at `lib/katello/repository_types/python.rb` and you can see how additional_metadata is defined. That whole hash is stored in the database as jsonb. For `python_packages` it has those two values, for `ostree_refs` it's `nil`.

Reply:
awesome, will update. Thanks for this.